.github/workflows/deploy.yml

```diff
@@ -42,7 +42,10 @@ jobs:
           bun run sst secret set StackProjectId ${{ secrets.STACK_PROJECT_ID }} --stage production
           bun run sst secret set StackPublishableClientKey ${{ secrets.STACK_PUBLISHABLE_CLIENT_KEY }} --stage production
           bun run sst secret set StackSecretServerKey ${{ secrets.STACK_SECRET_SERVER_KEY }} --stage production
-          bun run sst secret set GroqApiKey ${{ secrets.GROQ_API_KEY }} --stage production
+          bun run sst secret set BedrockRoleArn ${{ secrets.BEDROCK_ROLE_ARN }} --stage production
```

Comment:
Can SST create this automatically instead of external injection?

Reply:
Building on our Friday discussion, OK to keep Bedrock as independently deployed service (not via SST). We need to do that anyway for other providers (Vertex, Voyage). It can be argued it's cleaner to have it consistent across all providers.